bin/make_view

```diff
@@ -96,4 +96,24 @@ CREATE VIEW combined_contributions AS
   LEFT OUTER JOIN oakland_candidates
   ON "FPPC"::varchar = "Filer_ID"
   WHERE "Start_Date" IS NULL OR "Tran_Date" >= "Start_Date";
+
+DROP VIEW IF EXISTS combined_independent_expenditures;
+CREATE VIEW combined_independent_expenditures AS
+  SELECT "FPPC" AS "Filer_ID", "Filer_NamL", "Exp_Date", "Sup_Opp_Cd", "Amount"
+  FROM (
+    SELECT "Filer_NamL", "Exp_Date", "Cand_NamF", "Cand_NamL", "Amount", "Sup_Opp_Cd"
```

Comment:
Since this only supports candidate IE's maybe that should be reflected in the name -- `candidate_combined_independent_expenditures` or something similar.

Though, I think the same calculation logic would also work for ballot measure IE's, right?